src/Service/ElasticaService.php

```diff
@@ -470,6 +471,85 @@ private function createElasticaSearch(Search $search, array $options): ElasticaS
         return $esSearch;
     }
 
+    /**
+     * @return string[]
+     */
+    private function getIndices(Search $search): array
+    {
+        if (0 === \count($search->getContentTypes()) && null === $search->getRegex()) {
+            return $search->getIndices();
+        }
+
+        $filteredIndices = [];
+        foreach ($this->getIndicesForContentTypes($search->getIndices()) as $contentType => $indices) {
+            if (!\in_array($contentType, $search->getContentTypes(), true) && \count($search->getContentTypes()) > 0) {
+                continue;
+            }
+
+            if (null === $search->getRegex()) {
+                $filteredIndices = \array_merge($filteredIndices, $indices);
+                continue;
+            }
+
+            foreach ($indices as $index) {
+                if (\preg_match(\sprintf('/%s/', $search->getRegex()), $index)) {
+                    $filteredIndices[] = $index;
+                }
+            }
+        }
+
+        return \array_unique($filteredIndices);
+    }
+
+    /**
+     * @param string[] $aliases
+     *
+     * @return array<string, array<int, string>>
```

Comment:
```suggestion
     * @return array<string, string[]>
```

Reply:
For some strange reason phpstan was saying it was an int ...